packages/meditrak-server/src/routes/importSurveys/importSurveys.js

```diff
@@ -193,21 +193,20 @@ export async function importSurveys(req, res) {
         // Work out what fields of the survey should be updated based on query params
         const fieldsToForceUpdate = {};
         if (req.query.countryIds) {
-          // Set the countries this survey is available in
           fieldsToForceUpdate.country_ids = getArrayQueryParameter(req.query.countryIds);
         }
         if (surveyGroup) {
-          // Set the survey group this survey is attached to
           fieldsToForceUpdate.survey_group_id = surveyGroup.id;
         }
         if (req.query.permissionGroup) {
-          // A non-default permission group was provided
```

Comment:
Comments like this seem a bit self-explanatory, but if someone has put them there I don't see any harm in keeping them (the `non-default` term here could give someone easy mental access to what's going on)

Reply:
I beg to differ but I'm super happy to leave this as is!

Reply:
Keen to hear your reasoning - I guess you find they clutter things and you get distracted by them? For me it depends on whether there's anything conveyed by the comment other than "set x" that a junior dev/open source dev/someone new to the codebase could get something out of that they might take longer to understand by purely reading the code. For e.g. I would definitely agree the next comment down is useless (`Set or update the code for this survey`), but find there might be value in the one about `countries this survey is available in` because it says a bit more about what `country_ids` actually does. Agree it wouldn't justify a comment if you were writing the code fresh, but just that we maybe lose a little something by removing them. Extremely minor point that I've already spent too many words on, so very happy either way!

Reply:
I agree with you that a couple of those comments are useful! I guess maintainers thought that they should follow the existing pattern and added comments that are not particularly useful - that's what I wanted to avoid. This may lead someone to ignore comments that are actually useful - happened to me when I deleted all those comments! 😛 

Let's keep the ones that seem to convey the non-obvious then! 🙌

Reply:
Perfect! Yes haha I agree, we shouldn't require consistency if that introduces noise